pom.xml

```diff
@@ -1198,7 +1198,9 @@
                 <groupId>org.hsqldb</groupId>
                 <artifactId>hsqldb</artifactId>
                 <version>${hsqldb.version}</version>
+                <!--
```

Comment:
I think we just can remove this instead of commenting out. Will stay in the git history.